device-types/Dell/PowerEdge-R750xs.yml

```diff
@@ -0,0 +1,81 @@
+---
+manufacturer: Dell
+model: PowerEdge R750xs
+slug: dell-poweredge-r750xs
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
+comments: |
+  [PowerEdge R750xs](https://dl.dell.com/content/manual49765834-dell-emc-poweredge-r750xs-manuel-d-installation-et-de-maintenance.pdf?language=fr-fr)
+rear-ports:
+  - name: Rear VGA
+    type: other
+  - name: Rear USB 2
+    type: other
+  - name: Rear USB 3
+    type: other
+  - name: iDrac
+    type: other
+front-ports:
+  - name: Front VGA
+    type: other
+    rear_port: Rear VGA
+  - name: Front USB 2
+    type: other
+    rear_port: Rear USB 2
+  - name: Front USB micro-AB
+    type: other
+    rear_port: iDrac
```

Comment:
This should be modelled under the `console-ports` section with the proper interface type